packages/components/src/components/form/IconButton/IconButton.tsx

```diff
@@ -16,6 +16,8 @@ export const IconButton: FC<IconButtonProps> = forwardRef<IconButtonProps, 'butt
 
     const Icon = React.cloneElement(icon, { size: responsiveSize });
 
+    console.warn('Deprecated: The component "IconButton" is deprecated. Please use "ButtonIcon" instead.');
```

Comment:
nit: can we add a @deprecated comment to the component so the editor warns if we try to use it and add a comment to the storybook too?